addons/mysql/templates/componentdefinition-mysql-5.7-orc.yaml

```diff
@@ -0,0 +1,206 @@
+      spec:
+        ports:
+          - name: mysql
+            port: 3306
+            targetPort: mysql
+  configs:
+    - name: mysql-replication-config
+      templateRef: oracle-mysql8.0-config-template
+      constraintRef: oracle-mysql8.0-config-constraints
+      volumeName: mysql-config
+      namespace: {{ .Release.Namespace }}
+  scripts:
+    - name: orc-mysql-scripts
+      templateRef: orc-mysql-scripts
+      namespace: {{ .Release.Namespace }}
+      volumeName: scripts
+      defaultMode: 0555
+  volumes:
+    - name: data
+      needSnapshot: true
+  systemAccounts:
+    - name: root
+      initAccount: true
+      passwordGenerationPolicy:
+        length: 10
+        numDigits: 5
+        numSymbols: 0
+        letterCase: MixedCases
+  lifecycleActions:
+    postProvision:
+      customHandler:
+        image: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:8.0.33
+        exec:
+          command:
+            - bash
+            - -c
+            - "/scripts/mysql-orchestrator-register.sh;"
+        preCondition: ComponentReady
+    preTerminate:
+      customHandler:
+        image: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:8.0.33
+        exec:
+          command:
+            - bash
+            - -c
+            - "/scripts/orchestrator-client -c forget -i ${KB_CLUSTER_NAME};"
+        preCondition: Immediately
+  runtime:
+    initContainers:
+      - image: {{ .Values.image.registry | default "docker.io" }}/mysql_audit_log:{{ .Values.image.tag }}
+        name: init-data
+        command:
+          - bash
+          - -c
+          - |
+            mkdir -p {{ .Values.dataMountPath }}/{log,binlog,auditlog}
+            cp /usr/lib/mysql/plugin/ {{ .Values.dataMountPath }}/plugin -r
+        imagePullPolicy: Always
+        volumeMounts:
+          - mountPath: {{ .Values.dataMountPath }}
+            name: data
+      - command:
+          - cp
+          - -r
+          - /usr/local/bin/curl
+          - /kubeblocks/curl
+        image: docker.io/shakefu/curl-static:latest
+        imagePullPolicy: Always
+        name: init-curl
+        volumeMounts:
+          - mountPath: /kubeblocks
+            name: kubeblocks
+    containers:
+      - name: mysql
+        image: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: IfNotPresent
+        command:
+          - bash
+          - -c
+          - |
+            mv {{ .Values.dataMountPath }}/plugin/audit_log.so /usr/lib64/mysql/plugin/
+            rm -rf {{ .Values.dataMountPath }}/plugin
+            chown -R mysql:root {{ .Values.dataMountPath }}
+            REPORT_HOST=MYSQL_ORDINAL_HOST_${KB_POD_NAME##*-}
+            /scripts/docker-entrypoint-8.0.sh mysqld --server-id $(( ${KB_POD_NAME##*-} + 1)) --report_host ${!REPORT_HOST} \
+            --plugin-load-add=rpl_semi_sync_source=semisync_source.so \
```

Comment:
sync_source/replica for mysql 8.0.x and i guess it should be sync_master/slave for mysql 5.7.x